jsonnet/telemeter-client.libsonnet

```diff
@@ -31,6 +31,14 @@ function(params) {
   secret: tc.telemeterClient.secret,
   servingCertsCABundle: tc.telemeterClient.servingCertsCABundle,
   deployment: tc.telemeterClient.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/part-of': 'openshift-monitoring',
```

Comment:
IMO we should only add these labels, and `'app.kubernetes.io/part-of'` should be propagated from the main config as in the other comment on thanos-querier.
The rest of the labels should be added to telemeter directly, but this can be done in a follow-up effort.

Reply:
This suggested change require addition of 'commonLabels' to the the upstream project telemeter. WDYT ?

Reply:
Yes, I think that would be the best approach. We would only be keeping `managed-by` here. But again, this can be done in a follow-up effort.

Reply:
PTAL https://github.com/openshift/telemeter/pull/385